parsing.js

```diff
@@ -149,6 +189,9 @@ function parseIntro(document) {
     return intro;
 }
 
+// What is the point of looping through
```

Comment:
I believe we return once we've found the end of the intro.
There could be multiple sections. For example:
1. Theme and subject matter
2. Names of the surah
3. Reasons for Revelation
4. Historical Context
5. etc...

In other words, the intro is comprised of multiple intro sections. You'll notice we do return a value after we've hit a horizontal line... but this can be subject to change.